github.js

```diff
@@ -71,7 +74,17 @@ var GithubLocation = function(options, ui) {
     throw 'Git not installed. You can install git from `http://git-scm.com/downloads`.';
   }
 
-  this.strictSSL = 'strictSSL' in options ? options.strictSSL : true;
+  this.defaultOptions = {
+    strictSSL: 'strictSSL' in options ? options.strictSSL : true
+  };
+
+  for (var i = 0, keys = ["ca", "cert", "key"]; i < keys.length; i++) {
+    var key = keys[i];
+    if (key in options) {
+      this.defaultOptions[key] = fs.readFileSync(expandTilde(options[key]), 'ascii');
```

Comment:
`ascii` is correct for PEM format, but I reread the request documentation and saw they also supported PFX/PKC12 format which will be binary. I've since removed any encoding specification per the examples in request.